ppafm/common.py

```diff
@@ -17,56 +19,61 @@
 AUMASS = 1.66053904020e-27  # [kg]
 
 
-# fmt: off
-# default parameters of simulation
-params = {
-    "PBC": True,
-    "nPBC": np.array([1, 1, 1]),
-    "gridN": np.array([-1, -1, -1]).astype(int),
-    "gridO": np.array([0.0, 0.0, 0.0]),
-    "gridA": np.array([0.0, 0.0, 0.0]),
-    "gridB": np.array([0.0, 0.0, 0.0]),
-    "gridC": np.array([0.0, 0.0, 0.0]),
-    "FFgrid0": np.array([-1.0, -1.0, -1.0]),
-    "FFgridA": np.array([-1.0, -1.0, -1.0]),
-    "FFgridB": np.array([-1.0, -1.0, -1.0]),
-    "FFgridC": np.array([-1.0, -1.0, -1.0]),
-    "moleculeShift": np.array([0.0, 0.0, 0.0]),
-    "probeType": "O",
-    "charge": 0.00,
-    "Apauli": 18.0,
-    "Bpauli": 1.0,
-    "ffModel": "LJ",
-    "Rcore": 0.7,
-    "r0Probe": np.array([0.00, 0.00, 4.00]),
-    "stiffness": np.array([-1.0, -1.0, -1.0]),
-    "klat": 0.5,
-    "krad": 20.00,
-    "tip": "s",
-    "sigma": 0.7,
-    "scanStep": np.array([0.10, 0.10, 0.10]),
-    "scanMin": np.array([0.0, 0.0, 5.0]),
-    "scanMax": np.array([20.0, 20.0, 8.0]),
-    "scanTilt": np.array([0.0, 0.0, -0.1]),
-    "tiltedScan": False,
-    "kCantilever": 1800.0,
-    "f0Cantilever": 30300.0,
-    "Amplitude": 1.0,
-    "plotSliceFrom": 16,
-    "plotSliceTo": 22,
-    "plotSliceBy": 1,
-    "imageInterpolation": "bicubic",
-    "colorscale": "gray",
-    "colorscale_kpfm": "seismic",
-    "ddisp": 0.05,
-    "aMorse": -1.6,
-    "tip_base": np.array(["None", 0.00]),
-    "Rtip": 30.0,
-    "permit": 0.00552634959,
-    "vdWDampKind": 2,
-    "#": None,
-}
-# fmt: on
+class PpafmParameters(pydantic.BaseModel):
+    """Parameters for the PPAFM simulation.
+
+    The parameters description can be found on the wiki page: https://github.com/Probe-Particle/ppafm/wiki/Setting-simulation-parameters
+    """
+
+    PBC: bool = True
```

Comment:
Should we use snake_case for all of these? Now it's a mixture of upper, lower, snake, and CamelCase.

Reply:
that would be too much for this PR, I would leave this point for a separate issue, if possible.

Reply:
Makes sense.